unison-cli/src/Unison/Auth/HTTPClient.hs

```diff
@@ -1,29 +1,27 @@
-module Unison.Auth.HTTPClient (newAuthorizedHTTPClient, AuthorizedHttpClient (..)) where
+module Unison.Auth.HTTPClient (newAuthenticatedHTTPClient, AuthenticatedHttpClient (..)) where
 
 import qualified Data.Text.Encoding as Text
 import Network.HTTP.Client (Request)
 import qualified Network.HTTP.Client as HTTP
 import qualified Network.HTTP.Client.TLS as HTTP
-import Unison.Auth.CredentialManager (CredentialManager)
-import Unison.Auth.Tokens (TokenProvider, newTokenProvider)
+import Unison.Auth.Tokens (TokenProvider)
 import Unison.Codebase.Editor.UCMVersion (UCMVersion)
 import Unison.Prelude
 import Unison.Share.Types (codeserverIdFromURI)
 import qualified Unison.Util.HTTP as HTTP
 
 -- | Newtype to delineate HTTP Managers with access-token logic.
-newtype AuthorizedHttpClient = AuthorizedHttpClient HTTP.Manager
+newtype AuthenticatedHttpClient = AuthenticatedHttpClient HTTP.Manager
```

Comment:
Sorry for the little bit of PR noise, but I realize I used the wrong "Auth*" name here 😶